Common/GPU/thin3d.h

```diff
@@ -614,6 +621,11 @@ struct DeviceCaps {
 	// Old style, for older GL or Direct3D 9.
 	u32 clipPlanesSupported;
 
+	// Presentation caps
+	int presentMaxInterval; // 1 on many backends
+	bool presentInstantModeChange; // Our VulkanContext doesn't currently support it so we mark it as such, but it can be supported with careful coding.
```

Comment:
This feels like a comment that will one day become outdated but also stay here for a long time.  There's already a comment in Vulkan...

-[Unknown]